### PR TITLE
Edit Assignment1-Traindata

### PR DESCRIPTION
**Pull request by sylviapearce**
Reorg and copyedit Traindata portion of Assignment 1.

@MarkHoeber I'm not sure how I managed to get these checkpoint files in here, and I've also somehow copied in a trace from where I ran a cell and got an error message. But at least you can see the organizational changes here. Please let me know what you think.

### CODE REVIEW

File: Assignment1/Assignment1-Traindata.ipynb [hunk 1]
```diff
@@ -127,18 +117,18 @@
    "outputs": [],
    "source": [
     "fig = plt.figure()\n",
-    "fig.suptitle(\"NYC Taxi Trip Duration\")\n",
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: "taxi" showed up in a few places—I changed them all to "train"

File: Assignment1/Assignment1-Traindata.ipynb [hunk 2]
```diff
@@ -127,18 +117,18 @@
...
+    "fig.suptitle(\"NYC Train Trip Duration\")\n",
     "ax = fig.add_subplot(111)\n",
     "duration = train['trip_duration']\n",
-    "# create a variable with the number of observations in duration\n",
+    "# Create a variable with the number of observations in the duration.\n",
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: I'm not sure what the "observations" here refer to. The number of trains that are observed? The number of people on each train?

File: Assignment1/Assignment1-Traindata.ipynb [hunk 3]
```diff
@@ -149,34 +139,34 @@
...
     "            arrowprops=dict(facecolor='black', shrink=0.05))\n",
-    "# whisker 2\n",
+    "# Whisker 2.\n",
     "ax.annotate(str(two)+'th percentile', xy=(1.05, perc(two)), xytext=(1.25, perc(two)),\n",
     "            arrowprops=dict(facecolor='black', shrink=0.05))\n",
-    "# add y axis\n",
+    "# Add the y axis.\n",
     "plt.ylabel('Duration (seconds)')\n",
     "\n",
     "#---------------Enter your code here---------------#\n",
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: For this code cell, we don't tell the students to change anything. Do we need to?

File: Assignment1/Assignment1-Traindata.ipynb [hunk 4]
```diff
@@ -229,52 +225,52 @@
...
-    "# whisker 1\n",
+    "# Whisker 1.\n",
     "ax.annotate(str(one)+'th percentile', xy=(1.05, perc(one)), xytext=(1.25, perc(one)),\n",
     "            arrowprops=dict(facecolor='black', shrink=0.05))\n",
-    "# whisker 2\n",
+    "# Whisker 2.\n",
     "ax.annotate(str(two)+'th percentile', xy=(1.05, perc(two)), xytext=(1.25, perc(two)),\n",
     "            arrowprops=dict(facecolor='black', shrink=0.05))\n",
     "plt.ylabel(\"Duration (seconds)\")\n",
     "#---------------Enter your code here---------------#\n",
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: In this code cell, we don't tell the students to change anything beforehand. Should we? Do they need to enter their own code here?

File: Assignment1/Assignment1-Traindata.ipynb [hunk 5]
```diff
@@ -447,47 +431,71 @@
...
+   "execution_count": 1,
    "metadata": {
     "attributes": {
      "classes": [],
      "id": "",
      "n": "50"
     }
    },
-   "outputs": [],
+   "outputs": [
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: This is the error I got when I hit "Run" by mistake here. I wasn't sure how to remove it from the file or PR.

File: Assignment1/Assignment1-Traindata.ipynb [hunk 6]
```diff
@@ -512,21 +522,16 @@
...
-    "# Answer code\n",
-    "We used the following code in the cell above:\n",
+    "#### Python Notes\n",
     "\n",
-    "```python\n",
-    "#-------------Enter your code here-------------------------------#\n",
-    "plt.hist(np.log(train['trip_duration']), bins='auto', color='black') \n",
-    "#----------------------------------------------------------------#\n",
-    "```"
+    "Taking the log may make the values easier to interpret in the final graph."
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: I'm not sure what this refers to/means.

File: Assignment1/Assignment1-Traindata.ipynb [hunk 7]
```diff
@@ -512,21 +522,16 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Most likely trip duration\n",
+    "#### Most likely trip duration\n",
```
Comment (sylviapearce) [inline Assignment1/Assignment1-Traindata.ipynb]: I'm not sure what the purpose of this exercise is. If the histogram already shows the most likely trip duration, why are we adding it to a formula and running a cell? Is this a number the user is supposed to choose at random?

File: Assignment1/Assignment1-Traindata.ipynb (hunk 1 above)
Comment (geofflangenderfer) [reply Assignment1/Assignment1-Traindata.ipynb]: train.csv is a file filled with taxi trip data. It's confusing, but train refers to a partition of all of the data between test/train sets.

**Comment by MarkHoeber** [comment]
@sylviapearce  I fixed validation error, and removed extra file

@geofflangenderfer can we merge and update course lab?

**Pull request merged**